crates/bevy_ui/src/node_bundles.rs

```diff
@@ -75,7 +75,7 @@ impl Default for NodeBundle {
 pub struct ImageBundle {
     /// Describes the logical size of the node
     ///
-    /// This field is automatically managed by the UI layout system.
+    /// This component is automatically managed by the UI layout system.
     /// To alter the position of the `NodeBundle`, use the properties of the [`Style`] component.
```

Comment:
There's also a copy of this comment here, associated with the `node` field, with the same copy/paste mistake.

Seems safe to get rid of this one too to me. Not every bundle has it. But maybe it was intended that it say "to alter the **size**?